eth/chains/base.py

```diff
@@ -553,10 +557,14 @@ def estimate_gas(self, transaction: BaseTransaction, at_header: BlockHeader=None
         with self.get_vm(at_header).state_in_temp_block() as state:
             return self.gas_estimator(state, transaction)
 
-    def import_block(self, block: BaseBlock, perform_validation: bool=True) -> BaseBlock:
+    def import_block(self,
+                     block: BaseBlock,
+                     perform_validation: bool=True
+                     ) -> Tuple[Tuple[BaseBlock, ...], Tuple[BaseBlock, ...]]:
         """
         Imports a complete block.
```

Comment:
When you come back in here will you update this docstring to indicate what the two return values represent.